pkg/apis/application/v1alpha1/app_catalog_entry_types.go

```diff
@@ -95,6 +95,8 @@ type AppCatalogEntrySpecCatalog struct {
 
 // +k8s:openapi-gen=true
 type AppCatalogEntrySpecChart struct {
+	// ApiVersion is the The Helm chart API version.
+	ApiVersion string `json:"apiVersion"`
```

Comment:
```suggestion
	APIVersion string `json:"apiVersion"`
```

Good to have it under chart. 👍 Let's just make API upper case.